README.md

```diff
@@ -1733,7 +1733,7 @@ A [search engine](https://en.wikipedia.org/wiki/Search_engine_(computing)) is an
 
 **[`^        back to top        ^`](#)**
 
-
+- [Hydrazine](https://github.com/jareer12/hydrazine) - Create Discord bots without any coding, just drag and drop. `CC-BY-NC 4.0` `Typescript`
```

Comment:
Congratulations on the contribution!

This belongs in [`non-free.md`](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/non-free.md). Id place this under the Software Development section. 

- The language should also be changed to `Nodejs`

Reply:
Non-free? But Hydrazine is a free-to-use project.

Also, the central part of the Software is Typescript(Frontend React), not Node.js.

Reply:
The license is non-free https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/non-free.md#list-of-licenses

Node is needed to install the software - https://github.com/jareer12/hydrazine/blob/main/docs/HOW_TO_RUN.MD#run-with-ts-node